sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/util/SchemaNoPersistence.java

```diff
@@ -0,0 +1,35 @@
+package io.stargate.sgv2.restsvc.util;
+
+import io.stargate.db.schema.ImmutableKeyspace;
+import io.stargate.db.schema.Keyspace;
+import io.stargate.db.schema.Schema;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Bogus {@link Schema} implementation that we need for Stargate V2 as we do not have actual
+ * Persistence implementation. This means that keyspace and table entities are constructed on-demand
+ * without validation for existence.
+ */
+public class SchemaNoPersistence extends Schema {
+  private static final Keyspace ANONYMOUS = ImmutableKeyspace.builder().name("<anonymous>").build();
+
+  @Override
+  public Set<Keyspace> keyspaces() {
+    return Collections.emptySet();
+  }
+
+  @Override
+  public Keyspace keyspace(String name) {
+    if (name == null) {
+      return ANONYMOUS;
```

Comment:
what's the deal with anonymous?

Reply:
Copied from original `Schema`: not sure what the intent there is. Could try just leaving it out, or create dynamically.

Reply:
No idea, someone else should comment here I guess..

Reply:
Does it mean a query without a keyspace? `SELECT * FROM tableName` vs `SELECT * FROM keyspaceName.tableName`? I'm not sure that even makes sense in the context of the REST API where the resource requires a valid keyspace.

Reply:
How about I'll remove it from here since all usage here should be for proper keyspace and `null` should be invalid.